### DISCUSSION TITLE
Can jspsych-html-button-response-button be disabled?

### DISCUSSION THREAD

**Discussion by einGlasRotwein** [jspsych/jsPsych#1416] [Q&A] [answered]
Hey everyone,

I am trying to disable a html response button (and enable it after some time has passed) from the html-button-response plugin.

Hiding a response button works:
`document.getElementById('jspsych-html-button-response-button-0').style.visibility = "hidden";`

However, disabling it doesn't have an effect:
`document.getElementById('jspsych-html-button-response-button-0').disabled = true;`

Can those buttons be disabled? And if so, how?
Thank you in advance!

**Comment by jodeleeuw** [comment, marked as answer]
Hi @einGlasRotwein,

Yes, you can disable the buttons. The reason your code doesn't work is that you are selecting the `<div>` element that contains the `<button>`. To access the `disabled` property you have to select the button itself. This should work:

`document.querySelector('#jspsych-html-button-response-button-0 button').disabled = true;`

**Reply by jodeleeuw** [reply]
Edited my response above because it needs to use `querySelector()` instead of `getElementById()`

**Reply by einGlasRotwein** [reply]
Oh, I see, that makes sense, thank you!
